src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,590 @@
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'src/python/WMCore/WMRuntime/Unpacker.py')
+        else:
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'WMCore/WMRuntime/Unpacker.py')
+
+        self.agent = getattr(config.Agent, 'agentName', 'WMAgent')
+        self.sandbox = None
+        self.scriptFile = None
+        self.defaultTaskPriority = getattr(config.BossAir, 'defaultTaskPriority', 0)
+        self.maxTaskPriority = getattr(config.BossAir, 'maxTaskPriority', 1e7)
+        self.jobsPerSubmit = getattr(config.JobSubmitter, 'jobsPerSubmit', 200)
+
+        # Required for global pool accounting
+        self.acctGroup = getattr(config.BossAir, 'acctGroup', "production")
+        self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
+
+        # Build a requirement string
+        self.reqStr = "stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS=?=\"any\") || (GLIDEIN_REQUIRED_OS=?=REQUIRED_OS)) && (TARGET.Cpus >= RequestCpus)"
+        if hasattr(config.BossAir, 'condorRequirementsString'):
+            self.reqStr = config.BossAir.condorRequirementsString
+
+        # x509 proxy handling
+        proxy = Proxy({'logger': myThread.logger})
+        self.x509userproxy = proxy.getProxyFilename()
+        self.x509userproxysubject = proxy.getSubject()
+
+        return
+
+    def submit(self, jobs, info=None):
+        """
+        _submit_
+
+
+        Submit jobs for one subscription
+        """
+
+        # If we're here, then we have submitter components
+        self.scriptFile = self.config.JobSubmitter.submitScript
+
+        successfulJobs = []
+        failedJobs = []
+
+        if len(jobs) == 0:
+            # Then was have nothing to do
+            return successfulJobs, failedJobs
+
+        schedd = htcondor.Schedd()
+
+        # Submit the jobs
+        for jobsReady in grouper(jobs, self.jobsPerSubmit):
+
+            cluster_ad = self.getClusterAd()
+            proc_ads = self.getProcAds(jobsReady)
+
+            logging.debug("Start: Submitting %d jobs using Condor Python SubmitMany" % len(proc_ads))
+            try:
+
+                clusterId = schedd.submitMany(cluster_ad, proc_ads)
+
+            except Exception as ex:
+
+                logging.error("SimpleCondorPlugin job submission failed.")
+                logging.error("Moving on the the next batch of jobs and/or cycle....")
+                logging.exception(ex)
+
+                condorErrorReport = Report()
+                condorErrorReport.addError("JobSubmit", 61202, "CondorError", str(ex))
+                for job in jobsReady:
+                    job['fwjr'] = condorErrorReport
+                    failedJobs.append(job)
+
+                # Report submission error to operators
+                # TODO - Dirk: Do we really need this ?
```

Comment:
@ticoann , do we still need the alerts ? Weren't you deprecating or changing this ?